### DISCUSSION TITLE
Functionality of Duration but with date selection

### DISCUSSION THREAD

**Discussion by Ch1r4na** [kimai/kimai#4753] [General]
### Describe the issue

Hello Everyone,

We have the problem that we would like to work only with the duration, but unfortunately, we then cannot set the date. Is there a way to introduce this?

Many thanks.

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

2.0.22

### How do you run Kimai?

Virtual Server or alike

### Which PHP version are you using?

8.1

**Comment by kevinpapst** [comment]
Check System > Preferences > Timesheet mode
